iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -623,6 +623,20 @@ static Value allocateBufferForResult(OpBuilder &b, Operation *op,
   } else {
     return nullptr;
   }
+
+  // If its a static allocation hoist it all the begining of the function.
+  if (dynamicDims.empty()) {
+    auto parentOp = op->getParentOp();
```

Comment:
you can simplify all of this by 
```
auto funcOp = op->getParentOp<FuncOp>();
```

Reply:
oooo thats my loop but better :) thanks!